### PR TITLE
Puppet docs should accurately explain how to install puppet modules.

### PR DESCRIPTION
**Pull request by ndmckinley**
This is likely to be a huge stumbling block for users who are new to our gems.  I think this is an important fix.

-----------------------------------------------------------------
# [all]
## [puppet]
README updates for clarity re: installation.
### [puppet-storage]

### CODE REVIEW

File: products/_bundle/templates/puppet/README.md.erb [hunk 1]
```diff
@@ -49,6 +49,21 @@ Puppet module installer:
...
+you will also need to install those, with
+
+    /opt/puppetlabs/puppet/bin/gem install googleauth google-api-client
+
+If you prefer, you could also add the following to your puppet manifest:
+
+		package { [
+				'googleauth',
+				'google-api-client',
+			]:
```
Comment (danawillow) [inline products/_bundle/templates/puppet/README.md.erb]: nit: this alignment looks weird to me. The ] is lining up with the 'a' in package, which seems like probably not the right place for it (same in other files too)

File: products/_bundle/templates/puppet/README.md.erb [hunk 2]
```diff
@@ -49,6 +49,21 @@ Puppet module installer:
 
     puppet module install google/cloud
 
+Since these modules depend on the `googleauth` and `google-api-client` gems,
+you will also need to install those, with
+
+    /opt/puppetlabs/puppet/bin/gem install googleauth google-api-client
```
Comment (danawillow) [inline products/_bundle/templates/puppet/README.md.erb]: is `gem` always going to live in that directory?

File: products/_bundle/templates/puppet/README.md.erb (hunk 1 above)
Comment (ndmckinley) [reply products/_bundle/templates/puppet/README.md.erb]: I think it's correct - that's one more indent than `package`, which seems right to me.

File: products/_bundle/templates/puppet/README.md.erb (hunk 2 above)
Comment (ndmckinley) [reply products/_bundle/templates/puppet/README.md.erb]: Yep!

**Pull request merged**